Automattic/simplenote-android
# Issue/728 update sign out

### Fix
Update the ***Sign Out*** settings by removing the custom layout in order to match the layout of other settings and close #728.

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item.
3. Notice layout of settings.
3. Scroll to ***Account*** section.
4. Notice ***Sign out*** layout matches other settings.

### Review
Only one developer is required to review these changes, but anyone can perform the review.
